deploy-manage/deploy/cloud-enterprise/ece-configuring-ece-instance-configurations-edit.md

```diff
@@ -1,4 +1,7 @@
 ---
+applies_to:
+  deployment:
+    ece: all
```

Comment:
the steps on this page should link to the list of default instance configurations